stubs/algopy-stubs/itxn.pyi

```diff
@@ -61,6 +61,9 @@ _TResult_co = typing.TypeVar(
 )
 
 class _InnerTransaction(typing.Protocol[_TResult_co]):
+    def stage(self, *, begin_group: bool = False) -> None:
```

Comment:
All good as things currently stand, but if we do a release before this is merged, will need to bump stubs version again.

Reply:
compiler v5.5.0 + stubs v3.2.0 has been released so will need to bump stubs after a rebase